crates/allocator/src/lib.rs

```diff
@@ -22,15 +20,15 @@ pub fn panic(_info: &::core::panic::PanicInfo) -> ! {
 
 /// Need to provide an allocation error handler which just aborts
 /// the execution with trap.
-#[cfg(not(test))]
+#[cfg(no_std)]
 #[alloc_error_handler]
 #[no_mangle]
 fn oom(_: core::alloc::Layout) -> ! {
     ::core::intrinsics::abort();
 }
 
 /// Needed for non-wasm targets.
```

Comment:
Update the comment

Reply:
Comments are from here: https://github.com/rustwasm/wee_alloc/blob/master/example/src/lib.rs